docs/DeveloperGuide.md

```diff
@@ -175,6 +176,29 @@ Step 7: `AddCommand` then returns a new `CommandResult` with the result of the e
 
 ![Sequence diagram for the Add Command](images/AddSequenceDiagram.png)
 
+### Edit feature
+
+#### Implementation
+
+The Edit mechanism is facilitated by `EditCommand` and `EditCommandParser`. It allows users to edit a contact from their contact list and specify which attribute of the contacts' `Name`, `Phone`, `Email`, `Birthday`, `Address`, and `Tags` to be changed with the `Person` contact created.
+
+#### Example Usage
+
+Step 1: The user inputs `edit 1 p/91234567 e/johndoe@example.com`. This the phone number and email address of the person with an index of 1 to be `91234567` and `johndoe@example.com` respectively.
+
+Step 2: `LogicManager` calls `AddressBookParser#parseCommand` with the user input.
+
+Step 3: `AddressBookParser` will parse the command word and create a new `EditCommandParser` and call its function `parse` with the index as the arguments.
+
+Step 4: The `EditCommandParser#parse` will then parse the arguments and create a new `EditCommand` object.
+
+Step 5: The `LogicManager` then calls `EditCommand#execute`.
+
+Step 6: The `EditCommand` communicates with the `Model` to add the person by calling `Model#editPerson`.
+
+Step 7: `EditCommand` then returns a new `CommandResult` with the result of the execution.
+
+![Sequence diagram for the Edit Command](images/EditSequenceDiagram.png)
```

Comment:
I think you could add the limitation of PlantUML that causes the lifeline to continue after the 'X'.

Reply:
Thank you for bringing that up!